### DISCUSSION TITLE
Welcome to dimpact-zaakafhandelcomponent Discussions!

### DISCUSSION THREAD

**Discussion by edgarvonk** [infonl/dimpact-zaakafhandelcomponent#1348] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bas-info-nl** [comment]
My name is Bas and I am working on the same team as Edgar to help build the Zaakafhandelcomponent. I don't have a very active development role, but am always interested in ideas and am happy to answer questions where I can.
